.github/ISSUE_TEMPLATE/template-incubation-application.md

```diff
@@ -0,0 +1,251 @@
+
+## Contributors and Community
+
+Note: this section may be augmented by the completion of a Governance Review from TAG Contributor Strategy.
+
+### Suggested
+
+- [ ] **Contributor ladder with multiple roles for contributors.**
+
+<!-- (Project assertion goes here) --> 
+
+### Required
+
+- [ ] **Clearly defined and discoverable process to submit issues or changes.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Project must have, and document, at least one public communications channel for users and/or contributors.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **List and document all project communication channels, including subprojects (mail list/slack/etc.).  List any non-public communications channels and what their special purpose is.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Up-to-date public meeting schedulers and/or integration with CNCF calendar.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Documentation of how to contribute, with increasing detail as the project matures.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Demonstrate contributor activity and recruitment.**
+
+<!-- (Project assertion goes here) --> 
+
+## Engineering Principles
+
+### Suggested
+
+- [ ] **Roadmap change process is documented.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **History of regular, quality releases.**
+
+<!-- (Project assertion goes here) --> 
+
+### Required 
+
+- [ ] **Document project goals and objectives that illustrate the project’s differentiation in the Cloud Native landscape as well as outlines how this project fulfills an outstanding need and/or solves a problem differently.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document what the project does, and why it does it - including viable cloud native use cases.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document and maintain a public roadmap or other forward looking planning document or tracking mechanism.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document overview of project architecture and software design that demonstrates viable cloud native use cases, as part of the project's documentation.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document the project's release process.**
+
+<!-- (Project assertion goes here) --> 
+
+## Security
+
+Note: this section may be augemented by a joint-assessment performed by TAG Security.
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Clearly defined and discoverable process to report security issues.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Enforcing Access Control Rules to secure the code base against attacks (Example: two factor authentication enforcement, and/or use of ACL tools.)**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document assignment of security response roles and how reports are handled.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document Security Self-Assessment.**
```

Comment:
Any template?

Reply:
TAG Security's template is the current one, it may change in the future

Reply:
you can see this via https://contribute.cncf.io/maintainers/security/security-guidelines/#3-securitymd